### DISCUSSION TITLE
更新之后Set Speech Language没有数据？

### DISCUSSION THREAD

**Discussion by hiwens** [lencx/ChatGPT#563] [Q&A]
0.12.0刚更新替换的老版，看见可以设置这个 Speech Language，但是进入无法选择，显示no date 如何解决？

**Comment by lencx** [comment]
麻烦提供一下操作系统，你可以打开浏览器控制台（`Menu -> Help -> Toggle Developer Tools`），执行以下代码：

```js
speechSynthesis.getVoices();
```

查看有没有数据输出。

**Reply by hiwens** [reply]
系统是win11，执行后仍然是No date

**Reply by lencx** [reply]
这个功能是借助浏览器自身的 `SpeechSynthesis` API，文本到语音引擎（来自 chatgpt 的回答）：
- Windows：在控制面板 > 程序和功能中检查是否安装了 “Microsoft Speech Engine” 或 “Microsoft Speech Platform”。
- Mac：在“系统偏好设置” > “辅助功能” > “语音” > “系统语音”中检查是否安装了 “Alex” 或 “Fred” 语音。
- Linux：通过在终端中运行 “espeak” 或 “festival” 命令来检查是否安装了 “espeak” 或 “festival” 软件包。

**Comment by Ethkuil** [comment]
我一开始也是这样，重启了一下程序就正常了。
